Error when using an indirect fixture

Hi, I get an error while trying to use pytest-lazy-fixtures with indirect fixtures parametrization.

Here is a minimal reproducible example:
```python
import pytest
from pytest_lazy_fixtures import lf, lfc


@pytest.fixture(params=["Alessio"])
def username(request):
    return request.param


@pytest.mark.parametrize(
    "filters",
    [
        {
            "users": [
                lfc(lambda v: v.upper(), lf("username")),
                lfc(lambda v: v.lower(), lf("username")),
            ]
        },
    ],
)
def test_smth(filters):
    assert filters == {"users": ["ALESSIO", "alessio"]}
```

Actual output:
```
============================================================================ test session starts =============================================================================
platform darwin -- Python 3.12.9, pytest-8.3.5, pluggy-1.5.0 -- /bin/python
cachedir: .pytest_cache
configfile: pyproject.toml
plugins: lazy-fixtures-1.1.2
collected 0 items / 1 error                                                                                                                                                  

=================================================================================== ERRORS ===================================================================================
_______________________________________________________ ERROR collecting test_foo.py _______________________________________________________
python3.12/site-packages/pluggy/_hooks.py:513: in __call__
    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)
python3.12/site-packages/pluggy/_manager.py:120: in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
python3.12/site-packages/_pytest/python.py:245: in pytest_pycollect_makeitem
    return list(collector._genfunctions(name, obj))
python3.12/site-packages/_pytest/python.py:462: in _genfunctions
    self.ihook.pytest_generate_tests.call_extra(methods, dict(metafunc=metafunc))
python3.12/site-packages/pluggy/_hooks.py:574: in call_extra
    return self._hookexec(self.name, hookimpls, kwargs, firstresult)
python3.12/site-packages/pluggy/_manager.py:120: in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
python3.12/site-packages/pytest_lazy_fixtures/plugin.py:24: in pytest_generate_tests
    normalize_metafunc_calls(metafunc)
python3.12/site-packages/pytest_lazy_fixtures/normalizer.py:50: in normalize_metafunc_calls
    calls = _normalize_call(callspec, metafunc, used_keys)
python3.12/site-packages/pytest_lazy_fixtures/normalizer.py:81: in _normalize_call
    fm.pytest_generate_tests(newmetafunc)
python3.12/site-packages/_pytest/fixtures.py:1610: in pytest_generate_tests
    metafunc.parametrize(
python3.12/site-packages/_pytest/python.py:1304: in parametrize
    newcallspec = callspec.setmulti(
python3.12/site-packages/_pytest/python.py:1059: in setmulti
    raise ValueError(f"duplicate parametrization of {arg!r}")
E   ValueError: duplicate parametrization of 'username'
============================================================================== warnings summary ==============================================================================
python3.12/site-packages/_pytest/python.py:462
  python3.12/site-packages/_pytest/python.py:462: PluggyTeardownRaisedWarning: A plugin raised an exception during an old-style hookwrapper teardown.
  Plugin: pytest_lazyfixture, Hook: pytest_generate_tests
  ValueError: duplicate parametrization of 'username'
  For more information see https://pluggy.readthedocs.io/en/stable/api_reference.html#pluggy.PluggyTeardownRaisedWarning
    self.ihook.pytest_generate_tests.call_extra(methods, dict(metafunc=metafunc))

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
========================================================================== short test summary info ===========================================================================
ERROR test_foo.py - ValueError: duplicate parametrization of 'username'
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
======================================================================== 1 warning, 1 error in 0.60s =========================================================================

```

--- a/pytest_lazy_fixtures/normalizer.py
+++ b/pytest_lazy_fixtures/normalizer.py
@@ -72,7 +72,7 @@ def _normalize_call(callspec, metafunc, used_keys):
         fixturenames_closure, arg2fixturedefs = _get_fixturenames_closure_and_arg2fixturedefs(fm, metafunc, value)
 
         if fixturenames_closure and arg2fixturedefs:
-            extra_fixturenames = [fname for fname in fixturenames_closure if fname not in params]
+            extra_fixturenames = [fname for fname in set(fixturenames_closure) if fname not in params]
 
             newmetafunc = _copy_metafunc(metafunc)
             newmetafunc.fixturenames = extra_fixturenames

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -83,3 +83,8 @@ def fixture_a() -> str:
 @pytest.fixture
 def fixture_b() -> str:
     return "b"
+
+
+@pytest.fixture(params=["Alessio"])
+def username(request):
+    return request.param

--- a/tests/tests.py
+++ b/tests/tests.py
@@ -168,3 +168,18 @@ def test_foo(fixture2):
 )
 def test_dict_a_b(test_dict):
     assert test_dict == {"a": "b"}
+
+
+@pytest.mark.parametrize(
+    "filters",
+    [
+        {
+            "users": [
+                lfc(lambda v: v.upper(), lf("username")),
+                lfc(lambda v: v.lower(), lf("username")),
+            ]
+        },
+    ],
+)
+def test_smth(filters):
+    assert filters == {"users": ["ALESSIO", "alessio"]}

